.changeset/rude-clouds-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
Oh right, realized this should be `patch` since this is a non-breaking change and the package is on 0.x

Reply:
Can I just modify that file to change `minor` to `patch` or should I re-run the whole changeset command? Not sure if that has other side effects.

Reply:
you can just modify the file

Reply:
Fixed with amended commit.